care/facility/api/serializers/prescription.py

```diff
@@ -22,18 +22,18 @@ class Meta:
 class PrescriptionSerializer(serializers.ModelSerializer):
     id = serializers.UUIDField(source="external_id", read_only=True)
     prescribed_by = UserBaseMinimumSerializer(read_only=True)
-    last_administered_on = serializers.SerializerMethodField()
+    last_administration = serializers.SerializerMethodField()
     medicine_object = MedibaseMedicineSerializer(read_only=True, source="medicine")
     medicine = serializers.UUIDField(write_only=True)
 
-    def get_last_administered_on(self, obj):
+    def get_last_administration(self, obj):
         last_administration = (
             MedicineAdministration.objects.filter(prescription=obj)
             .order_by("-administered_date")
             .first()
         )
         if last_administration:
-            return last_administration.administered_date
+            return LastAdministrationSerializer(last_administration).data
```

Comment:
Any reason for a dedicated serializer to serialize last administration?

This can be a `MedicineAdministrationSerializer` itself right?